app/controllers/users_controller.rb

```diff
@@ -7,19 +7,8 @@ class UsersController < ApplicationController
                except: %i[index signout_confirm add_org_admin remove_org_admin remove_from_org confirm_destroy]
   before_action :initialize_stripe, only: %i[edit]
 
-  INDEX_ATTRIBUTES_FOR_SERIALIZATION = %i[id name username summary profile_image].freeze
-  private_constant :INDEX_ATTRIBUTES_FOR_SERIALIZATION
-
   def index
-    @users =
-      case params[:state]
-      when "follow_suggestions"
-        determine_follow_suggestions(current_user)
-      when "sidebar_suggestions"
-        Users::SuggestForSidebar.call(current_user, params[:tag]).sample(3)
-      else
-        User.none
-      end
+    @users = sidebar_suggestions || User.none
```

Comment:
It's starting to feel less like this belongs in this controller at all - possible follow-up could be to move `sidebar_suggestions` into a separate controller, but that feels like low-priority effort to me (and a big distraction from everything that's happening in _this_ PR).